ddtrace/tracer/span.go

```diff
@@ -122,8 +131,14 @@ func (s *span) setTagError(value interface{}, debugStack bool) {
 		s.Error = 1
 		s.Meta[ext.ErrorMsg] = v.Error()
 		s.Meta[ext.ErrorType] = reflect.TypeOf(v).String()
-		if debugStack {
-			s.Meta[ext.ErrorStack] = string(debug.Stack())
+		if !cfg.noDebugStack {
+			var stack string
```

Comment:
Lets just have:
```go
if cfg.stackFrames = 0 {
    s.Meta[ext.ErrorStack] = string(debug.Stack())
} else {
    s.Meta[ext.ErrorStack] = takeStacktrace(cfg.stackFrames, cfg.stackSkip)
}
```
Without the extra variable.

Reply:
Fixed.